extensions/amp-render/1.0/component.js

```diff
@@ -100,7 +107,7 @@ export function RenderWithRef(
     [refresh]
   );
 
-  const rendered = useRenderer(render, data);
+  const rendered = useRenderer(data === DEFAULT_DATA ? null : render, data);
```

Comment:
Could you add a comment explaining why we do this comparison, and then pass the renderer fn as `null`?